statsmodels/iolib/table.py

```diff
@@ -878,7 +878,8 @@ def set_datatype(self, val):
     stub=r'\textbf{%s}',
     empty='',
     missing='--',
-    replacements={"%" : "\%", ">" : "$>$", "|" : "$|$"}
+    #replacements will be processed in lexicographical order
```

Comment:
@tvanessa  just to understand this
Is there a reason for processing in lexicographical order, and to add sorting?
or is it just to make it independent of how the list is filled?